test/LoggerAbstractServiceFactoryTest.php

```diff
@@ -291,7 +291,7 @@ public function testWillInjectWriterPluginManagerIfAvailable()
     public function testWillInjectProcessorPluginManagerIfAvailable()
     {
         $processors = new ProcessorPluginManager(new ServiceManager());
-        $mockProcessor = $this->getMock('Zend\Log\Processor\ProcessorInterface');
+        $mockProcessor = $this->getMockBuilder('Zend\Log\Processor\ProcessorInterface')->getMock();
```

Comment:
as above, can be just `$this->createMock('Zend\Log\Processor\ProcessorInterface');`